packages/replay-orchestrator/src/test-run/parallel-tests/run-all-tests.ts

```diff
@@ -402,3 +418,18 @@ const getTestCasesWithBaseTestRunId = async ({
   }
   return testsToRun;
 };
+
+const shouldUseBaseTestRunTestCases = async ({
+  client,
+  baseTestRunId,
+}: {
+  client: AxiosInstance;
+  baseTestRunId: string;
+}) => {
+  const testRun = await getTestRun({ client, testRunId: baseTestRunId });
```

Comment:
Haven't we already loaded this, stored as fallbackTestRun?

Reply:
🤦 Yep, https://github.com/alwaysmeticulous/meticulous-sdk/pull/340/commits/8c90537f00c991b576784068958c6314190cc0e6 drops the extra call